lib/google_drive/spreadsheet.rb

```diff
@@ -134,7 +134,24 @@ def batch_update(requests)
 
     # Append values to a spreadsheet by first searching for a data table at a range,
     # then appending the specified values at the end of this data table.
-    def append(range_name, values, override_params = {})
+    #
+    # +range+ The A1 notation of a range to search for a logical table of data. 
+    # Values will be appended after the last row of the table.
+    # +values+ Array (rows) of Array (columns) of values to append to the spreadsheet.
+    # +override_params+ allows you to control how the values will be inserted.
```

Comment:
Can you be more specific what kind of params are available, maybe by referring to a specific section of the Google's API doc? Can we specify "Query parameters" in the doc here?:
https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values/append#query-parameters